lib/cisco_node_utils/bgp_af.rb

```diff
@@ -264,8 +258,7 @@ def default_advertise_l2vpn_evpn
 
     # dampen_igp_metric
     def dampen_igp_metric
-      result = config_get('bgp_af', 'dampen_igp_metric', @get_args)
-      result ? result.first.to_i : nil
+      config_get('bgp_af', 'dampen_igp_metric', @get_args)
```

Comment:
This property has the auto_default tag.  From my understanding, we should keep the old code since we can't automatically handle the default value?

```
dampen_igp_metric:
  auto_default: false
  kind: int
  config_get_token_append: '/^dampen-igp-metric (\d+)$/'
  config_set_append: '<state> dampen-igp-metric <num>'
  default_value: 600
```